### ISSUE TITLE
Blockstar: Full Site Editor - font sizes do not update 

### ISSUE THREAD

**Issue by brittanygray902** [Automattic/wp-calypso#94124]
### Quick summary

User is having trouble adjusting font to smaller size. When you use the Typography options and use either the slider for resizing or the set sizes, small, medium, large, etc. the text is not responding and resizing. I notice that instead of the text resizing it looks like it's resizing the space / margin of I believe, the column block instead. 

### Steps to reproduce

1. Opened user's Editor & edited the "navigation menu items" size through typography settings. (Navigation is actually just paragraph blocks that are linked) 
2. Can replicate issue with sizing not responding. Instead I am seeing the column or maybe group block is being affected. I am also getting "The editor encountered an unexpected error." when attempting to adjust these paragraph blocks / text. 

What I see when I attempt to resize paragraph / linked text: 

https://github.com/user-attachments/assets/d15746ca-3a3f-472d-b635-558d4eb89fa5

Other steps tried:
- On Premium plan test site I set the Blockstar theme & pasted their Editor content there. Could not replicate issues with resizing the text on my test site. 

### What you expected to happen

I expect the text to resize when using the typography settings, for each paragraph block. Do not expect to see the error message either. 

### What actually happened

When trying to resize paragraph block / text, I end up getting this error: "The editor encountered an unexpected error." 

I've included the copied error message: 

```
Error: Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at qr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:74962)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76204)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76666)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76666)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76469)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76666)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
```

Refreshing does remove the editor error though and I can continue to manage the template in editor. 
    
What I then see when I attempt to resize paragraph / linked text: 

https://github.com/user-attachments/assets/d15746ca-3a3f-472d-b635-558d4eb89fa5

### Impact

One

### Available workarounds?

No but the platform is still usable

### If the above answer is "Yes...", outline the workaround.

Maybe CSS could fix this but I haven't been able to create any that will easily override the text sizing. Potentially a CSS wizard could help recommend some to fix this as a work around for now? 

### Platform (Simple and/or Atomic)

Simple

### Logs or notes

Theme: Blockstar

Editor error:

```
Error: Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at qr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:74962)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76204)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76666)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76666)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76469)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
    at Xr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76666)
    at Yr (https://s0.wp.com/wp-content/plugins/gutenberg-core/v19.1.0/build/vendors/react-dom.min.js?m=1724868497i&ver=18:1:76354)
```

**Comment by brittanygray902** [comment]
Forgot to add the interaction for reference. ZD-8669629

**Comment by jartes** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Replicated
- Tested on Atomic – Replicated

📌 FINDINGS
- I'm able to replicate this with other themes, and this issue is related to #74234
- In this case, this user will need to set to default the size for links in global styles to get the paragraph block font size being applied.

📌 ACTIONS
- Triaged
- Closed as duplicate #74234

📌 Message to Author
Note for the author or another commenter—important if requesting author feedback or clarification.
